fragile/optimize/env.py

```diff
@@ -138,6 +140,25 @@ def reset(self, batch_size: int = 1, **kwargs) -> StatesEnv:
         )
         return new_states
 
+    def calculate_custom_end(self, points: numpy.ndarray, end: numpy.ndarray) -> numpy.ndarray:
+        """
+        Check if points already in the bounds, are not in a custom domain \
+        defined by out_of_domain_check(point) == True. \
+        The base implementation returns False as a default.
+
+        Args:
+            points: points to be checked.
+
+        Returns:
+            Boolean array containing the out condition for the points.
+
+        """
+        if self.out_of_domain_check != None:
```

Comment:
To check against `None` we use the expression `is None` or `is not None` instead of `==` or `!=`.
```suggestion
        if self.out_of_domain_check is not None:
```

Reply:
Yeah, I just fixed it 1s after PR, I did not "make check" prior to PR and it gave me this advice too.

Reply:
When a comment is dealt with / outdated, you can click resolve conversation to hide it.